pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -310,7 +312,11 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 					return NE, []byte("!~")
 				}
 			default:
-				tkn.setErr(`expected "=" after "!", got "%c" (%d)`, tkn.lastChar, tkn.lastChar)
+				if isValidCharAfterOperator(tkn.lastChar) {
+					return Not, tkn.bytes()
+				}
+
```

Comment:
Can you please remove this empty line? I should maybe link to our [style guide](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines) (point 3) somewhere in this repo too. Until then, I'll have to annoy you with my comments.

Reply:
😁 . Of course, blank line removed ✅ .